docker-entrypoint.sh

```
@@ -1,12 +1,23 @@
 #!/bin/sh
-set -eu
-# Enable pipefail only when supported
-( set -o pipefail ) 2>/dev/null && set -o pipefail
+set -euo pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)